lib/generator/baseGenerator.py

```diff
@@ -86,6 +86,33 @@ def init_result_dict_variable(self, default_number_list, default_list_list):
             init_result_dict[init_list_key] = []
         return init_result_dict
 
+    def generate_update_result_for_combination(self, input_update_result, input_compare_result, input_run_time_dict):
```

Comment:
The implementation of `generate_update_result_for_combination()` method should be put in sub-class `InputLatencyFrameThroughputGenerator`.